cpp/src/Slice/Parser.h

```diff
@@ -754,7 +761,7 @@ class InterfaceDef : public virtual Container, public virtual Contained
 public:
 
     void destroy() override;
-    OperationPtr createOperation(const std::string&, const TypePtr&, bool, int, Operation::Mode = Operation::Normal);
+    OperationPtr createOperation(const std::string&, const TaggedDefListTokPtr&, Operation::Mode = Operation::Normal);
```

Comment:
The "grammar" could simply create the operation without the return values, and then add each return value, just like it adds parameters.

Reply:
Yes, I think that this would be better than passing it into the constructor. Changing it now!